gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/events/SimulationFault.java

```diff
@@ -0,0 +1,41 @@
+package gov.pnnl.goss.gridappsd.dto.events;
+
+import java.util.Map;
+
+import gov.pnnl.goss.gridappsd.dto.events.Fault.FaultImpedance;
+import gov.pnnl.goss.gridappsd.dto.events.Fault.PhaseCode;
+import gov.pnnl.goss.gridappsd.dto.events.Fault.PhaseConnectedFaultKind;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+public class SimulationFault extends Event{
```

Comment:
Use either this or Fault class which is already here. It should be called just Fault and not SimulationFault.

Reply:
Looks like you are using things from Fault class. Just copy them here and remove Fault class. After that rename this class as Fault.